### DISCUSSION TITLE
Das ZUGFeRD-PDF ist valide.  -> Signatur: unknown

### DISCUSSION THREAD

**Discussion by m4Viper** [ZUGFeRD/mustangproject#613] [General]
Hello everybody,

validating my created ZUGFeRD PDF says -> pdf valide, but shows the message: "signature: unknown"

Needs my zugferd-pdf a signature or can this message be ignored?

**Comment by jstaerk** [comment]
Hi,
no, not at all, signature of the authoring software, that is for statitical purposes only and could e.g. ghostscript in case there are clues that that was used as the Factur-X attaching software.

kind regards
Jochen
